app/lib/utils/extensions.dart

```diff
@@ -22,3 +22,14 @@ extension ThemeExtension on BuildContext {
       MediaQuery.platformBrightnessOf(this) == Brightness.dark;
   bool get isKeyboardVisible => MediaQuery.viewInsetsOf(this).bottom > 0;
 }
+
+extension ColorOpacity on Color {
+  Color adjustOpacity(double opacity) {
```

Comment:
oh, I see what you did here. This is good, but the recommendation from Flutter 3.27 is to use `withValues` ?

Reply:
Ah. Yes with values(alpha:). updated and removed extension